client-js/app/scripts/app.ts

```diff
@@ -141,13 +141,20 @@ window.addEventListener('WebComponentsReady', () => {
 
   class RegistryActivity implements Activity {
     path: Array<string>;
+    private globElem: LabradRegistry;
 
-    constructor(path: Array<string>) {
+    private api: registry.RegistryServiceJsonRpc;
```

Comment:
So I'm getting a type signature error when I use ```RegistryApi``` instead. The code in ```DatasetActivity``` uses ```DataVaultService``` instead of ```DataVaultApi```